recipes/libui/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from conans import ConanFile, CMake, tools
+from conans.tools import os_info
+import os
+
+
+class libuiConan(ConanFile):
+    name = "libui"
+    description = "Simple and portable GUI library in C that uses the native GUI technologies of each platform it supports."
+    topics = ("conan", "libui", "ui", "gui")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/andlabs/libui"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "libui-"+os.path.basename(
+            self.conan_data["sources"][self.version]['url']).removesuffix('.tar.gz')
+        os.rename(extracted_dir, self._source_subfolder)
```

Comment:
I wonder if this new feature `strip_root` could clean this up

```suggestion
        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Reply:
@grafikrobot GTK got rebuild (#5262). Could you please retrigger a build, by either close for ~10 secs then reopen this PR or merge something like this code suggestion?